templates/signup.html

```diff
@@ -104,6 +104,45 @@ <h4 class="text-blue-500 mt-0 mb-0 text-center">{{_('create_student_account_expl
                         {% endfor %}
                     </select>
                 </div>
+                <!-- START OF NEW QUESTION -->
+                {% if is_teacher %}
+                <div class="signup-container">
+                    <label class="signup-label" for="heard_about">{{_('heard_about_hedy')}}</label>
+                    <ul class="signup-input appearance-none">
+                        <li class="flex" style="justify-content: end;">
+                            <div class="flex flex-row items-center gap-2">
+                                <input type="checkbox" name="heard_about" value="from_another_teacher">
+                                <label style="width: 201px;">{{_('from_another_teacher')}}</label>
+                            </div>
+                        </li>
+                        <li class="flex" style="justify-content: end;">
+                            <div class="flex flex-row items-center gap-2">
+                                <input type="checkbox" name="heard_about" value="social_media">
+                                <label style="width: 201px;">{{_('social_media')}}</label>
+                            </div>
+                        </li>
+                        <li class="flex" style="justify-content: end;">
+                            <div class="flex flex-row items-center gap-2">
+                                <input type="checkbox" name="heard_about" value="from_video">
+                                <label style="width: 201px;">{{_('from_video')}}</label>
+                            </div>
+                        </li>
+                        <li class="flex" style="justify-content: end;">
+                            <div class="flex flex-row items-center gap-2">
+                                <input type="checkbox" name="heard_about" value="from_magazine_website">
+                                <label style="width: 201px;">{{_('from_magazine_website')}}</label>
+                            </div>
+                        </li>
+                        <li class="flex" style="justify-content: end;">
+                            <div class="flex flex-row items-center gap-2">
+                                <input type="checkbox" name="heard_about" value="other_source">
+                                <label style="width: 201px;">{{_('other_source')}}</label>
+                            </div>
+                        </li>
```

Comment:
Do we want to enable users to select multiple options? Otherwise a radio button or a select dropdown would make more sense (and removes a lot of duplicate code).